src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/BasicResultProvider.vbproj

```diff
@@ -73,11 +73,6 @@
   <ItemGroup>
```

Comment:
Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.